src/services/profileComparisonLabelResolverStrategy/TargetHighestStrategy.ts

```diff
@@ -1,22 +1,28 @@
-import ProfileComparisonLabelResolverStrategy from "./ProfileComparisonLabelResolverStrategy"
+import ProfileComparisonLabelResolverStrategy from './ProfileComparisonLabelResolverStrategy'
 
 export default class TargetHighestStrategy extends ProfileComparisonLabelResolverStrategy {
-    getLabelsAndValues(mainDataSet: any, selfDataSet: any, maxNumberOfFields: number) {
-        let mainDataSetArray: Array<any> = []
-        let selfDataSetArray: Array<any> = []
-        Object.entries(mainDataSet).forEach((element) => {
-            mainDataSetArray.push(element)
-        })
-        // @ts-ignore
-        let processedMainDataSetArray = mainDataSetArray.sort((e1, e2) => e1[1] > e2[1]).slice(0, maxNumberOfFields)
-        processedMainDataSetArray.forEach((element, idx) => {
-            selfDataSetArray.push([element[0], (selfDataSet[element[0]] || 0)])
-        });
+  getLabelsAndValues(
+    mainDataSet: any,
+    selfDataSet: any,
+    maxNumberOfFields: number
+  ) {
+    const mainDataSetArray: Array<any> = []
+    const selfDataSetArray: Array<any> = []
+    Object.entries(mainDataSet).forEach((element) => {
+      mainDataSetArray.push(element)
+    })
+    // @ts-ignore
```

Comment:
this auto-formatting broke this @ts-ignore comment (the ts-ignore should apply to the line below, `sort((e1, e2) => e1[1] > e2[1])`